DEVELOPER.md

```diff
@@ -14,10 +14,13 @@ See http://kafka.apache.org/documentation.html#consumerconfigs for details about
 
     input {
         kafka {
-            topic_id => ... # string (required), The topic to consume messages from
+            topic_id => ... # string (optional), default: nil, The topic to consume messages from. Can be a java regular expression for whitelist of topics.
```

Comment:
There is not straightforward way of describing a set of parameters as mutually-exclusive and at-least-one-required.

topic_id || white_list || black_list is required.